tests/Resources/fixtures/empty-double.js

```diff
@@ -0,0 +1 @@
+module.exports = "";
```

Comment:
All JS files should have a newline at the end. Not sure what editor you use, but for example Atom enforces this on save for me.

Reply:
Hmm, will double check my settings in vscode